doc/radosgw/multisite.rst

```diff
@@ -46,6 +46,24 @@ configurations for the Ceph Object Gateway:
   a global object namespace. This global object namespace ensures unique
   object IDs across zonegroups and zones.
 
+  It can be useful to create multiple zonegroups in cases where isolation of the
+  datasets is desirable when those datasets exist in a shared namespace of users
+  and buckets. It might be that you have several connected sites that share
```

Comment:
i think it's probably better to describe the residency/redirect concepts first to motivate the multi-zonegroup example. 'dataset' isn't very clear in this context, so i suggest using 'object data' instead

> Each bucket is owned by the zonegroup where it was created (except where overridden by the ``LocationConstraint`` (link to https://docs.ceph.com/en/latest/radosgw/placement/#s3-bucket-placement) on bucket creation), and its object data will only replicate to other zones in that zonegroup. Any request for data in that bucket that are sent to other zonegroups will redirect to the zonegroup where the bucket resides.
>
> It can be useful to create multiple zonegroups when you want to share a namespace of users and buckets across many zones, but isolate the object data to a subset of those zones. It might be that you have several connected sites that share storage, but only require a single backup for purposes of disaster recovery. In such a case, it could make sense to create several zonegroups with only two zones each to avoid replicating all objects to all zones.
>
> In other cases, it might make more sense to isolate things in separate realms, with each having a single zonegroup. Zonegroups provide flexibility by making it possible to control the isolation of data and metadata separately.

just below this is the `**Multiple Realms:**` section, so there may be a better way to lead into that

Reply:
I've changed "dataset" to "RADOS object data".

Reply:
> I've changed "dataset" to "RADOS object data".

did you read any of the other suggested changes? maybe it wasn't clear that i rewrote that whole quoted block

Reply:
@cbodley: See https://github.com/ceph/ceph/pull/52360.